zebra-chain/src/transparent/address.rs

```diff
@@ -223,6 +224,7 @@ impl Address {
         }
     }
 
+    #[allow(dead_code)]
```

Comment:
It's better to put attributes below comments: https://github.com/rust-lang/style-team/issues/72.